### PR TITLE
#49 added ASG aggregations

### PR DESCRIPTION
**Pull request by patrykorwat**

### CODE REVIEW

File: clouddiscovery/tests/provider/vpc/test_diagram.py [hunk 1]
```diff
@@ -186,3 +192,74 @@ def test_should_detect_public_and_private_subnet(self):
                 to_node=ResourceDigest(id=PRIVATE_SUBNET, type="aws_subnet"),
             )
         )
+
+    def test_asg_ecs(self):
```
Comment (meshuga) [inline clouddiscovery/tests/provider/vpc/test_diagram.py:196]: Codacy Issue found: [Method could be a function](https://app.codacy.com/gh/Cloud-Architects/cloudiscovery/pullRequest?prid=5692715)

**Comment by meshuga** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- clouddiscovery/shared/diagram.py  1
- clouddiscovery/provider/vpc/resource/compute.py  1
- clouddiscovery/provider/vpc/diagram.py  1
         

Complexity decreasing per file
==============================
+ clouddiscovery/shared/common.py  -1
         

Clones added
============
- clouddiscovery/tests/provider/vpc/test_diagram.py  6
- clouddiscovery/provider/vpc/diagram.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Cloud-Architects/cloudiscovery/pullRequest?prid=5692715&bid=18631402)

**Comment by meshuga** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- clouddiscovery/provider/vpc/diagram.py  1
- clouddiscovery/shared/diagram.py  1
- clouddiscovery/provider/vpc/resource/compute.py  1
         

Complexity decreasing per file
==============================
+ clouddiscovery/shared/common.py  -1
         

Clones added
============
- clouddiscovery/provider/vpc/diagram.py  2
- clouddiscovery/tests/provider/vpc/test_diagram.py  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Cloud-Architects/cloudiscovery/pullRequest?prid=5692715&bid=18631402)

File: clouddiscovery/tests/provider/vpc/test_diagram.py (hunk 1 above)
Comment (meshuga) [inline clouddiscovery/tests/provider/vpc/test_diagram.py:196]: Codacy Issue found: [Method could be a function](https://app.codacy.com/gh/Cloud-Architects/cloudiscovery/pullRequest?prid=5692715)

**Comment by meshuga** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- clouddiscovery/shared/diagram.py  1
- clouddiscovery/provider/vpc/diagram.py  1
- clouddiscovery/provider/vpc/resource/compute.py  1
         

Complexity decreasing per file
==============================
+ clouddiscovery/shared/common.py  -1
         

Clones added
============
- clouddiscovery/provider/vpc/diagram.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Cloud-Architects/cloudiscovery/pullRequest?prid=5692715&bid=18631402)

**Comment by meshuga** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- clouddiscovery/provider/vpc/resource/compute.py  1
- clouddiscovery/provider/vpc/diagram.py  1
- clouddiscovery/shared/diagram.py  1
         

Complexity decreasing per file
==============================
+ clouddiscovery/shared/common.py  -1
         

Clones added
============
- clouddiscovery/provider/vpc/diagram.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Cloud-Architects/cloudiscovery/pullRequest?prid=5692715&bid=18631402)

**Comment by meshuga** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- clouddiscovery/provider/vpc/diagram.py  1
- clouddiscovery/provider/vpc/resource/compute.py  1
- clouddiscovery/shared/diagram.py  1
         

Complexity decreasing per file
==============================
+ clouddiscovery/shared/common.py  -1
         

Clones added
============
- clouddiscovery/provider/vpc/diagram.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Cloud-Architects/cloudiscovery/pullRequest?prid=5692715&bid=18631402)

**Comment by meshuga** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- clouddiscovery/shared/diagram.py  1
- clouddiscovery/provider/vpc/resource/compute.py  1
- clouddiscovery/provider/vpc/diagram.py  1
         

Complexity decreasing per file
==============================
+ clouddiscovery/shared/common.py  -1
         

Clones added
============
- clouddiscovery/provider/vpc/diagram.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Cloud-Architects/cloudiscovery/pullRequest?prid=5692715&bid=18631402)

**Comment by patrykorwat** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- clouddiscovery/provider/vpc/diagram.py  1
- clouddiscovery/provider/vpc/resource/compute.py  1
- clouddiscovery/shared/diagram.py  1
         

Complexity decreasing per file
==============================
+ clouddiscovery/shared/common.py  -1
         

Clones added
============
- clouddiscovery/provider/vpc/diagram.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Cloud-Architects/cloudiscovery/pullRequest?prid=5692715&bid=18631402)

**Review by leandrodamascena — APPROVED** [review]

**Pull request merged**